src/sbmlutils/report/sbmlinfo.py

```diff
@@ -156,6 +156,41 @@ def info_sbase(sbase: libsbml.SBase) -> Dict[str, Any]:
                     replaced_elements_combined = "".join(replaced_elements)
                 info["replaced_elements"] = replaced_elements_combined
 
+        # distrib
+        sbml_distrib = sbase.getPlugin("distrib")
+        if sbml_distrib and type(sbml_distrib) == libsbml.DistribSBasePlugin:
+
+            # list to store uncertainities
+            info["uncertainties"] = []
+
+            listOfUncertainties = sbml_distrib.getListOfUncertainties()
```

Comment:
variables in python should be with underscore no camelcase, i.e. `list_of_uncertainties`. Better just directly iterate over the list. I.e. 

```
for uncertainty in sbml_distrib.getListOfUncertainties():
```